cobc/tree.h

```diff
@@ -1656,6 +1656,17 @@ struct list_error {
 	char			*msg;		/* Error Message text */
 };
 
+typedef struct s_line_file {
```

Comment:
as this is _only_ used in cobc.c, it should be moved there

Reply:
Ok, I placed that in cobc.c